src/registry.ts

```diff
@@ -335,6 +335,9 @@ function handleGetEntryErrResponse(err: AxiosError): SignedRegistryEntry {
   // Check if status was 404 "not found" and return null if so.
   if (err.response.status === 404) {
     return { entry: null, signature: null };
+  } else if (err.response.status >= 400) {
+    console.log(err);
```

Comment:
Why would you `console.log`? Is it a leftover debug line? This shows up in the stdout of anyone importing the SDK and can not be hidden, can get quite spammy.

Reply:
Stop commenting on old commits!! lol